examples/material-next-app-router-ts/next.config.js

```diff
@@ -0,0 +1,16 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  swcMinify: true,
+  modularizeImports: {
+    '@mui/icons-material': {
+      transform: '@mui/icons-material/{{member}}',
+    },
+  },
+  experimental: {
+    appDir: true,
```

Comment:
I think starting from `next@13.4` the `appDir` is already stable and doesn't need to be included in the `experimental` block

Reply:
@MartinXPN - Thank you corrected.